website/project/metadata/egap-registration.json

```diff
@@ -0,0 +1,418 @@
+	"version": 2,
+	"pages": [{
+			"id": "page1",
+			"title": "General Information About the Project",
+			"questions": [{
+					"qid": "q1",
+					"nav": "Title",
+					"type": "string",
+					"format": "text",
+					"title": "B1 Title of Project ",
+					"description": "Provide the working title of your study.",
+					"required": true
+				},
+				{
+					"qid": "q2",
+					"nav": "Authors",
+					"title": "B2 Authors",
+					"help": "Jimmy Stewart, Ava Gardner, Bob Hope, Greta Garbo",
+					"format": "textarea",
+					"required": true
+				},
+				{
+					"qid": "q3",
+					"nav": "EGAP Registration ID",
+					"title": "EGAP Registration ID",
+					"format": "textarea",
+					"required": true
+				},
+				{
+					"qid": "q4",
+					"nav": "Timestamp",
+					"title": "Timestamp of original registration",
+					"format": "textarea",
+					"required": true
+				},
+				{
+					"qid": "q5",
+					"nav": "Acknowledgements",
+					"title": "B3 Acknowledgements",
+					"type": "string",
+					"format": "textarea",
+					"required": false
+				},
+				{
+					"qid": "q6",
+					"title": "B4 Is one of the study authors a university faculty member?",
+					"nav": "University Faculty Member?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"Yes",
+						"No",
+						"Other"
+					],
+					"description": "Please choose one"
+				},
+				{
+					"qid": "q7",
+					"title": "Other",
+					"format": "textarea",
+					"required": false
+				},
+				{
+					"qid": "q8",
+					"nav": "Authors",
+					"title": "B2 Authors",
+					"help": "Jimmy Stewart, Ava Gardner, Bob Hope, Greta Garbo",
+					"format": "textarea",
+					"required": true
+				},
+				{
+					"qid": "q9",
+					"title": "B5 Is this Registration Prospective or Retrospective?",
+					"nav": "Prospective or Retrospective?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"Registration prior to any research activities",
+						"Registration prior to assignment of treatment",
+						"Registration prior to realization of outcomes",
+						"Registration prior to researcher access to outcome data",
+						"Registration prior to researcher analysis of outcome data",
+						"Registration after researcher analysis of outcome data",
+						"Other"
+					],
+					"description": "Please choose one"
+				},
+				{
+					"qid": "q10",
+					"title": "B6 Is this an experimental study?",
+					"nav": "Experimental study?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"Yes",
+						"No"
+					],
+					"description": "(with random assignment of units to different conditions)"
+				},
+				{
+					"qid": "q11",
+					"title": "B7 Date of start of study",
+					"nav": "Date of start of study",
+					"type": "string",
+					"format": "text",
+					"description": "Understood as first date of treatment assignment or equivalent for observational study",
```

Comment:
Where is this description coming from? Is it our text or coming from the original form that I can't see?